### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Creeperz653**
Idk

### CODE REVIEW

**Comment by BlueTree242** [comment]
Ok, let me check

**Comment by Creeperz653** [comment]
done?

**Comment by BlueTree242** [comment]
Screenshot_20211225_012033 either put a banner or remove it, also i think it says rating 0 so maybe wrong image?

**Comment by BlueTree242** [comment]
fix the java version. bungee and spigot require 9 but velocity require 11

File: README.md [hunk 1]
```diff
@@ -1,5 +1,62 @@
...
+
+  * group merging.
+  * hides /pl.
+* blacklist or whitelist tab commands.
+* Others 
+
+### Requirements
+
+* Java 11 (or higher) (for Velocity)
+* Java 8 (or higher) (bungee and spigot)
```
Comment (BlueTree242) [inline README.md:44]: java 9 not 8

**Comment by Creeperz653** [comment]
done kid

**Comment by BlueTree242** [comment]
will merge soon

**Comment by Creeperz653** [comment]
BRUH

**Comment by BlueTree242** [comment]
uppercase some words, i mean fix the punctiuation

**Comment by Creeperz653** [comment]
WHERE

**Comment by BlueTree242** [comment]
everywhere

**Comment by Creeperz653** [comment]
bitch

File: README.md [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
+  * group merging.
+  * hides /pl.
+* blacklist or whitelist tab commands.
+* Others 
+
+### Requirements
+
+* Java 11 (or higher) (for Velocity)
+* Java 9 (or higher) (bungee and spigot)
+* ProtocolLib(spigot)
```
Comment (BlueTree242) [inline README.md:45]: space between the bracket and protocolib pls, also same for one under it

Comment (Creeperz653) [reply README.md:45]: what?

Comment (BlueTree242) [reply README.md:45]: and uppercase spigot

Comment (Creeperz653) [reply README.md:45]: OH

Comment (Creeperz653) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -1,5 +1,62 @@
...
+## Introduction
+
+### Reasons
+
+* This plugin was made to replace any paid plugin hiding
+
+### Features
+
+* Group merging.
+* Hides /pl (/plugin).
```
Comment (BlueTree242) [inline README.md:37]: /plugins not /plugin

Comment (Creeperz653) [reply README.md:37]: ?

Comment (Creeperz653) [reply README.md:37]: uk that works as well
test it

Comment (BlueTree242) [reply README.md:37]: image

Comment (Creeperz653) [reply README.md]: DONNNNNNE

**Pull request merged**